spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskDeleteService.java

```diff
@@ -262,44 +264,7 @@ public void deleteTaskExecutions(Set<Long> taskExecutionIds) {
 		auditData.put("Deleted Job Execution IDs", StringUtils.collectionToDelimitedString(jobExecutionIds, ", "));
 
 		if (!jobExecutionIds.isEmpty()) {
```

Comment:
@blueaadvark wdyt about pulling the call to `getTaskExecutionDeleteChunkSize(this.dataSource)` up here from https://github.com/spring-cloud/spring-cloud-dataflow/pull/5025/files#diff-ec79e846595460e3b9bcc697304fcee7dc73a8fa171c9ba48dacc21eda82986cR280 and then we can pass into this newly added method to avoid calling it 2x?

Reply:
I went ahead and took care of this doing merge.